### DISCUSSION TITLE
JavaScript unable to stop Screen share

### DISCUSSION THREAD

**Discussion by rajamarko** [ant-media/Ant-Media-Server#6846] [Q&A]
"@antmedia/webrtc_adaptor": "^2.11.3"

Steps to reproduce: 

1. start screen share with command  `webRTCAdaptor.switchDesktopCapture(streamId);`
2. select "Chrome Tab" and click button share
3. switch back to the camera with command webRTCAdaptor.switchVideoCameraCapture(streamId, deviceId);

Video output will be switched back to camera, but screen share is never stopped.

If on step 2. you select "Window", or "Entire Screen", step 3. will stop screen share

Browser is: Chrome Version 131.0.6778.86

**Comment by lastpeony** [comment]
This looks like a good bug finding. i will take a look and create an issue

**Reply by zerostack0** [reply]
Issue: https://github.com/ant-media/Ant-Media-Server/issues/6850
